source/docs/casper/developers/writing-onchain-code/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Introduction
-slug: /writing-contracts
+slug: /developers/writing-onchain-code/writing-contracts
```

Comment:
We should not change existing slugs, unless there is good reason.

```suggestion
slug: /writing-contracts
```

Reply:
This was changed because something else was linking to it, i believe.... I will remove and see if anything breaks